src/winetricks

```diff
@@ -6130,45 +6130,27 @@ w_metadata directmusic dlls \
     publisher="Microsoft" \
     year="2010" \
     media="download" \
-    file1="../directx9/directx_feb2010_redist.exe" \
-    installed_file1="$W_SYSTEM32_DLLS_WIN/dmusic.dll"
+    file1="../directx9/directx_feb2010_redist.exe"
 
 load_directmusic()
 {
 # Untested. Based off https://bugs.winehq.org/show_bug.cgi?id=4805 and https://bugs.winehq.org/show_bug.cgi?id=24911
 
```

Comment:
Please add a w_warn message here that the user may want to call individual verbs instead (ideally with an example)